editor/scene_tree_dock.cpp

```diff
@@ -2663,12 +2663,17 @@ void SceneTreeDock::_new_scene_from(String p_file) {
 	Node *base = selection.front()->get();
 
 	HashMap<const Node *, Node *> duplimap;
+	HashMap<const Node *, Node *> inverse_duplimap;
 	Node *copy = base->duplicate_from_editor(duplimap);
 
+	for (KeyValue item : duplimap) {
```

Comment:
```suggestion
	for (const KeyValue &item : duplimap) {
```
Although not sure if you can use `const` here`.

Reply:
The KeyValue here does not even use template.
I checked a few options and it doesn't compile. Not worth it.